sdk/resourcemanager/azure-resourcemanager-containerregistry/pom.xml

```diff
@@ -43,6 +43,7 @@
     </javaModulesSurefireArgLine>
 
     <parallelizeTests>false</parallelizeTests>
+    <revapi.skip>true</revapi.skip>
```

Comment:
Please add reason when adding `revapi.skip`.

Reply:
CI analysis exception occurs due to the existence of duplicate classes `IdentityPropertiesAutoGenerated` and `IdentityProperties`.